docs/reference/glossary.rst

```diff
@@ -0,0 +1,73 @@
+========
+Glossary
+========
+
+.. contents::
+   :local:
+   :backlinks: none
+
+.. We're not using the .. glossary:: admonition here, as we want to split the glossary terms into sections, which the tag does not support. If this ever changes, then we can add the tagging.
+
+Product Features
+================ 
+
+.. note:: All Flocker documentation will refer the `Production ready` product, unless tagged `Alpha` or `Beta`.
+
+Production Ready
+   This product or feature is suitable for production use. You can expect API stability between versions, robust security, and product stability for all supported uses.
```

Comment:
New sentence, new line. Here and other places. (search in the document for ". ").